lib/matplotlib/mpl-data/stylelib/classic.mplstyle

```diff
@@ -497,3 +497,5 @@ animation.convert_path: convert   # Path to ImageMagick's convert binary.
                                   # is also the name of a system tool.
 animation.convert_args:
 animation.html: none
+
+_internal.classic_mode: True
```

Comment:
There is a "no newline at end of file" marker here.  I've never seen that before.  Doesn't PEP8 require a newline at end of file?